iati-activities-schema.xsd

```diff
@@ -1085,14 +1079,7 @@
             <xsd:attribute name="ref" use="optional" type="xsd:string">
```

Comment:
Is it worth making a referenced attribute for this one??  It seems like this is defined identically multiple times??

Reply:
Possibly if everywhere `ref` is used has the same definition